packages/harmony/src/components/button/BaseButton/BaseButton.tsx

```diff
@@ -49,19 +47,56 @@ export const BaseButton = forwardRef<HTMLButtonElement, BaseButtonProps>(
 
     const ButtonComponent = asChild ? Slot : 'button'
 
+    const buttonComponentCss: CSSObject = {
+      fontFamily: 'var(--harmony-font-family)',
```

Comment:
Let's take this from the theme?

Reply:
don't think this exists in the theme currently? i can add though

Reply:
Oh I see it's not from typography?

Reply:
nice lets add it then, i think there is `fontByWeight` which im guessing we probably dont need anyway?

Reply:
Added 👍 , and yea i don't think that one would be used much